sklearn/multioutput.py

```diff
@@ -37,12 +39,90 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.copy(estimator)
+
+    if sample_weight is not None:
```

Comment:
I was wondering if 
```
estimator.partial_fit(X, y, classes=classes, sample_weight=sample_weight)
```
would suffice since ``partial_fit`` function of the the base estimator would handle it appropriately. Let me know if it sounds okay !